### PR TITLE
Add windows support

### PR DESCRIPTION
**Pull request by dkorel-copperleaf**
Adds basic support for a windows sensu client with a linux server backend. Most actions should be abstracted into their correct places... but testing thus far works.

### CODE REVIEW

**Review by sstaszkiewicz-copperleaf — COMMENTED** [review]
Filenames with spaces in them kinda suck.

**Review by sstaszkiewicz-copperleaf — COMMENTED** [review]
Looks like you included an empty file "tasks/Microsoft Windows Server 2012 R2 Standard/main.yml"

**Review by sstaszkiewicz-copperleaf — CHANGES_REQUESTED** [review]
Looks good. Remove the empty file and maaaaybe optionally change the directory name to not have spaces + add new lines at the end of all the files and good to merge.

**Comment by dkorel-copperleaf** [comment]
I know! Terribly ugly... the reason i left it like that was because they automatically detect the distro, and windows returns the OS with spaces...

**Comment by dkorel-copperleaf** [comment]
And that empty file is automatically called by the role - so it needs to be there. OS specific tasks could be added later . I believe the mechanism by which they detect the OS is something like $ansible_distribution - with windows boxes it's going to return spaces by default because that's just how they name the OS... 

Maybe we can use ansible_os_family which returns a single word of 'Windows'

File: defaults/main.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 ---
+
+# windows paths (forward slashes for JSON compatiblity)
+sensu_win_base_path: "c:/opt/sensu"
```
Comment (sstaszkiewicz-copperleaf) [inline defaults/main.yml:5]: I wonder if `{{ ansible_user_dir }}` works on windows. If it does, we could do `{{ (ansible_user_dir is defined)|ternary(ansible_user_dir+"/.sensu", "c:/opt/sensu") }}` or something. I dunno I'm just spitballing.

Comment (dkorel-copperleaf) [reply defaults/main.yml:5]: very fancy! The structure of the tasks is a little cluttered right now... everything is in the windows_client.yml file, but really should be broken out into in the distro specific locations

**Review by sstaszkiewicz-copperleaf — APPROVED** [review]

**Comment by lbussey** [comment]
I think this is ready for merge into master. Please confirm!

**Comment by gjohannsen-copperleaf** [comment]
Given the amount of changes, I think there could be issues but I will approve and we can merge it and keep it in mind to rollback if there any problems that arise in the next few days

**Review by gjohannsen-copperleaf — APPROVED** [review]

**Pull request merged**